adapters/flipp/flipp.go

```diff
@@ -223,3 +224,35 @@ func buildBid(decision *InlineModel, impId string) *openrtb2.Bid {
 	}
 	return bid
 }
+
+func paramsUserKeyPermitted(request *openrtb2.BidRequest) bool {
+	if request.Regs != nil {
+		if request.Regs.COPPA == 1 {
+			return false
+		}
+		if request.Regs.GDPR != nil && *request.Regs.GDPR == 1 {
+			return false
+		}
+	}
+	if request.Ext != nil {
+		var extData struct {
+			TransmitEids bool `json:"transmitEids"`
+		}
+		if err := json.Unmarshal(request.Ext, &extData); err == nil {
+			if !extData.TransmitEids {
+				return false
+			}
+		}
```

Comment:
should not ignore `UnMarshal` error. It could be a bad request error for  `request.Ext`

Reply:
For the purpose of this function `paramsUserKeyPermitted()`, if json unmarshal fails I would want to continue code execution for this function (i.e. move to the next if block `if request.User != nil && request.User.Consent != "" {}`. Am I thinking about that correctly?